lib/src/base-styles/_nx-lists.scss

```diff
@@ -133,104 +130,64 @@
   }
 
   .nx-list__item {
-    @include semi-bold();
-
-    color: $nx-text-color-light;
-
     &.selected {
-      background-color: $nx-grey-3;
-      color: $nx-text-color;
+      background-color: #d3e6ff;
       pointer-events: none;
```

Comment:
What's this for?

Reply:
> What's this for?

Removed here because I can't figure out why it's there: https://github.com/sonatype/sonatype-react-shared-components/pull/141/commits/8e5d1365db4c91fd78145c309fc151e35f771c4a